TESTS/mbedmicro-rtos-mbed/rtostimer/main.cpp

```diff
@@ -85,6 +85,11 @@ void error(const char* format, ...)
 {
     (void) format;
 }
+
+mbed_error_status_t mbed_error(mbed_error_status_t error_status, const char *error_msg, unsigned int error_value, const char *filename, int line_number) 
```

Comment:
From the comments above, it does not look like we need mbed_error here till we actually replace error with mbed_error in `EvrRtxTimerError`.

Reply:
Code will not be called, till we actual update `EvrRtxTimerError`. We can add this later when we are making changes to CMSIS/RTX code. As it helps in keeping track of all CMSIS/RTX changes in single PR and dependencies as well when updating to higher versions.